### DISCUSSION TITLE
Cannot immport stable osu into lazer

### DISCUSSION THREAD

**Discussion by TheSeaCat** [ppy/osu#19578] [Q&A] [answered]
I'm on macOS, and tried to import osu into lazer, but it seems like osu lazer kind of doesn't detect the file. I tried to rename it to .osu but i got an error message saying
<img width="243" alt="eeeeeeee" src="https://user-images.githubusercontent.com/110600407/182882967-4fb2545e-10c3-4dcd-9e49-5c3a746a74ff.png">
:

Here is a video with the glitch, ( the file size was too large to import )
https://www.youtube.com/watch?v=9iiXLb1QrJs

**Comment by frenzibyte** [comment, marked as answer]
You have to select the directory containing the osu!(stable) files, not the wine wrapper app. Should be `osu!.app/drive_c/osu!`.

**Reply by TheSeaCat** [reply]
That worked, ty!
